codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -843,4 +850,12 @@ public void setUseMultiAuth(boolean useMultiAuth) {
     public boolean useMultiAuth() {
         return useMultiAuth;
     }
+
+    public Map<String, ParameterModel> getEndpointParameters() {
```

Comment:
I'm not sure if this definition is enough as a criteria for triggering codegen of the JMESPath runtime helper. Endpoint parameters could in the future be present in the customization config without the need for that, since it's only operation context params with nested types that needs it. The ParameterModel does not currently contain information that lets you decide that either, but maybe when adding operation context params that can fit into ParameterModel.